lib/responder.js

```diff
@@ -13,21 +13,14 @@ var path = require('path');
 var awaitFinished = RSVP.denodeify(require('on-finished'));
 var destroy = require('destroy');
 
-// shink-ray is a fork of compression which supports advanced compression encodings but not ES5.
-var compressor;
-try {
-  compressor = require('shrink-ray')();
-} catch (e) {
-  compressor = require('compression')();
-}
-
 var Responder = function(req, res, options) {
   this.req = req;
   this.res = res;
   this.provider = options.provider;
   this.config = options.config || {};
   this.rewriters = options.rewriters || {};
-  this.compression = options.compression;
+  this.useCompression = (options.compression ? true : false);
```

Comment:
I don't think it's necessary. Relying on "truthy" values is generally okay in JS.